readme.md

```diff
@@ -312,50 +211,43 @@ yarn closure
 yarn compile-devtools
 ```
 
-## Lighthouse as trace processor
-
-Lighthouse can be used to analyze trace and performance data collected from other tools (like WebPageTest and ChromeDriver). The `traces` and `devtoolsLogs` artifact items can be provided using a string for the absolute path on disk. The devtoolsLogs array is captured from the Network domain (a la ChromeDriver's [`enableNetwork` option](https://sites.google.com/a/chromium.org/chromedriver/capabilities#TOC-perfLoggingPrefs-object)) and reformatted slightly. As an example, here's a trace-only run that's reporting on user timings and critical request chains:
-
-### `config.json`
-
-```json
-{
-  "audits": [
-    "user-timings",
-    "critical-request-chains"
-  ],
-
-  "artifacts": {
-    "traces": {
-      "defaultPass": "/User/me/lighthouse/lighthouse-core/test/fixtures/traces/trace-user-timings.json"
-    },
-    "devtoolsLogs": {
-      "defaultPass": "/User/me/lighthouse/lighthouse-core/test/fixtures/traces/perflog.json"
-    }
-  },
-
-  "aggregations": [{
-    "name": "Performance Metrics",
-    "description": "These encapsulate your app's performance.",
-    "scored": false,
-    "categorizable": false,
-    "items": [{
-      "audits": {
-        "user-timings": { "expectedValue": 0, "weight": 1 },
-        "critical-request-chains": { "expectedValue": 0, "weight": 1}
-      }
-    }]
-  }]
-}
-```
+## Docs & Recipes
+
+Useful documentation, examples, and recipes to get you started.
+
+**Docs**
+
+- [Using Lighthouse programmatically](./docs/readme.md#using-programmatically)
+- [Testing a site with authentication](./docs/readme.md#testing-on-a-site-with-authentication)
+- [Testing on a mobile device](./docs/readme.md#testing-on-a-mobile-device)
+- [Lighthouse Architecture](./docs/architecture.md)
 
-Then, run with: `lighthouse --config-path=config.json http://www.random.url`
+**Recipes**
 
-The traceviewer-based trace processor from [node-big-rig](https://github.com/GoogleChrome/node-big-rig/tree/master/lib) was forked into Lighthouse. Additionally, the [DevTools' Timeline Model](https://github.com/paulirish/devtools-timeline-model) is available as well. There may be advantages for using one model over another.
+- [gulp](docs/recipes/gulp) - helpful for CI integration
+- [Custom Audit example](./docs/recipes/custom-audit) - extend Lighthouse, run your own audits
+
+**Videos**
+
+The session from Google I/O 2017 covers architecture, writing custom audits,
+Github/Travis/CI integration, headless Chrome, and more:
+
+[![Lighthouse @ Google I/O](https://img.youtube.com/vi/NoRYn6gOtVo/0.jpg)](https://www.youtube.com/watch?v=NoRYn6gOtVo)
+
+_click to watch the video_
+
+## Related Projects
+
+* [webpack-lighthouse-plugin](https://github.com/addyosmani/webpack-lighthouse-plugin) -  run Lighthouse from a Webpack build.
```

Comment:
we should fix up the wording on these, too. "gather" vs "gathers", active vs passive, etc